tests/requirements.txt

```diff
@@ -1,3 +1,5 @@
+black
```

Comment:
Similarly, here, I think we should keep isort/black separate from running tests. People can run black and isort however they see fit. For example, maybe as a VS Code plug-in.